config/routes.rb

```diff
@@ -126,6 +126,24 @@
       to: "users#index",
       as: :users
 
+  resources :users,
+            only: [],
+            param: :id,
+            path: "/bo/users",
+            path_names: { show: "/:id" } do
+              resources :user_activations,
+                        as: :activations,
+                        only: %i[new create],
+                        path: "activate",
+                        path_names: { new: "" }
```

Comment:
Q: What do we need this `path_name` here for?

Reply:
I think it stops it tacking "new" on the end of the URL

Reply:
AAhhhh right! Yes I did have a vague record of why we used it but I wasn't sure :D We used it in the apps when we were dealing with the `identifiers` but we don't need it any more since tokens are a scoped route? I think we should keep it in this case, shall we?